src/routes/v1/task.route.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import { validateRequest } from '../../middleware/request-validator';
+import {
+  TaskCreateValidationSchema,
+  createTaskHandler
+} from '../../controllers/v1/task/create.controller';
+
+const taskRoute = Router();
+
+taskRoute.post('/', validateRequest('body', TaskCreateValidationSchema), createTaskHandler);
```

Comment:
I foresee problems with this pattern. Why not do this validation in the controller; that why you also retain the validated type